assembly/guide-configs/app-dev/python/project-setup.json

```diff
@@ -6,7 +6,7 @@
     "title": "Python SDK developer's guide - Project setup",
     "sidebar_label": "Project setup",
     "sidebar_position": 3,
-    "description": "The project setup section of the Temporal Python SDK Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application in Python—that is, all the relevant steps to start a Workflow Execution that executes an Activity.",
+    "description": "The project setup section of the Temporal Python SDK Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application in Python\u2014that is, all the relevant steps to start a Workflow Execution that executes an Activity.",
```

Comment:
🤔

Reply:
It renders the same - but let me delete it.